main/src/library/Option.flix

```diff
@@ -235,6 +235,11 @@ namespace Option {
         case Some(v) => f(v)
     }
 
+    pub def choose(l: Option[a], r: Option[a]): Option[a] = match l {
```

Comment:
This already exists. It's called "withDefault" or something like that.

Reply:
Ah, I must've missed it